tests/ci/integration_tests_runner.py

```diff
@@ -911,6 +929,13 @@ def run_impl(self, repo_path, build_path):
             group_counters, group_test_times, log_paths = self.try_run_test_group(
                 repo_path, group, tests, MAX_RETRY, NUM_WORKERS
             )
+
+            for fail_status in ("ERROR", "FAILED"):
```

Comment:
It would be really nice to not only match name of the test, but also failure reason/error code/whatever.

Reply:
I can look into that. Do we want to simply do `test_name:error_message` or try to copy Testflows syntax?